### PR TITLE
Adding illustration and some description about powergrid in readme

### PR DESCRIPTION
**Pull request by parulM1992**
Adding a folder named screens where I have put screenshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
-# Power Grid
-Power grid is an easy to use yet a powerful tool to readily build a cross-platform CSS Grid system.  
+# Powergrid
+
+![screen](screens/shot1.PNG)
+
```
Comment (parulM1992) [inline README.md]: Will be able to see this image after commit, so if required we change after verifying it.

Comment (MikeVaz) [reply README.md]: I think we can use some sort of service that would generate screens from the home page https://zs.github.com/powergrid

Comment (MikeVaz) [reply README.md]: E.g. https://www.url2png.com/

Comment (parulM1992) [reply README.md]: Tried this services as well as some other services also, but its not taking screenshots as required. And also most of them are paid services.

**Pull request merged**